src/main.cpp

```diff
@@ -450,6 +573,40 @@ class gfxPipelineHandler {
         UpdateWindow(hWND);
         SetFocus(hWND);
     }
+
+    static void Install() {
+        InstallPatch("Enables pointer in windowed mode.", { 0x90, 0x90 }, {
+            0x4F136E,
+        });
+
+        if (!datArgParser::Get("16bit")) {
+            InstallPatch("Creates the initial window in 32-bit mode.", { 32 }, {
+                0x401475,
+                0x401477,
+            });
+        }
+
+        InstallCallback("gfxPipeline::SetRes", "Enables extra resolution parameters (e.g. '-window')",
```

Comment:
I actually had to add them in order for the 32-bit thing to work properly.

How about we change the byte that passes "detectArgs" instead of forcing every one to detect args?